types.go

```diff
@@ -99,6 +99,12 @@ type Keys struct {
 	Private string `json:"private,omitempty"`
 }
 
+// Dump indicate information about an dump
```

Comment:
```suggestion
// Dump indicate information about an dump
//
// Documentation: https://docs.meilisearch.com/reference/api/dump.html
```

Reply:
Done...in the same commit but pushed with force 'cause I didn't want to "pollute" with an extra commit just for that.

Reply:
No problem, we squash and merge so in the end there is not much pollution, but thanks for the info! :)

Reply:
good to know

Reply:
Actually Bors creates a merge commit so the history looks like this:
https://github.com/meilisearch/meilisearch-go/network